labs/b4.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Beginner Lab 4 - Debian, packages, compiling software
+layout: lab
+---
+
+## What is a distribution?
+
+A Linux distribution is a combination of the Linux kernel, package management system, drivers, scheduler,
+ and other software. They are the operating systems that use Linux as a kernel. 
```

Comment:
there are non-Linux distributions. I'd write this as "combination of a kernel, drivers, libraries, and other critical programs necessary to have a functioning operating system" or something along those lines.

Reply:
I'd also add "distro for short"